internal/factsengine/factsengine_integration_test.go

```diff
@@ -1,6 +1,4 @@
-//go:build integration_test
-
-package integration_test
+package factsengine
```

Comment:
We have started in many places the `_test` suffice. We should continue this way here:
`package factsengine_test`

Reply:
@arbulu89 @CDimonaco should i move every other test in factsengine to the test package?

Reply:
If you want yes, It will help to keep consistency, If there are tests that cannot be moved (globals, testing of unexported thing etc.., just skip it, we will take them into another pr)

Reply:
I wouldn't do it in this PR @fabriziosestito 
Let's merge this as it is already reviewed